core/sql/ustat/hs_globals.cpp

```diff
@@ -4896,7 +4900,7 @@ void HSGlobalsClass::getMemoryRequirementsForOneGroup(HSColGroupStruct* group, I
 // on the amount of memory allocated to hold their values.
 //
 // Note: row count may be adjusted up if DP2 sampling and heavy amount of varchars.
-Int64 HSGlobalsClass::getInternalSortMemoryRequirements()
+Int64 HSGlobalsClass::getInternalSortMemoryRequirements(NABoolean performISForMC)
```

Comment:
Why is a parameter used here? Is this a "what if" call that sets the static global variable later?